provider/pkg/gen/go-templates/helm/v3/chart.tmpl

```diff
@@ -0,0 +1,312 @@
+// Copyright 2016-2020, Pulumi Corporation.
```

Comment:
Any particular reason why this file is templated? It doesn't seem to have any substitutions as far as I can tell. Just makes it a little more cumbersome to review. Same goes for the types defined below.

Reply:
IIRC, I did that to keep the Go linter from complaining. I think it was trying to analyze the templates in place and was failing since the paths are wrong unless the files are in the SDK folder. It's not necessary otherwise.